.golangci.yml

```diff
@@ -0,0 +1,23 @@
+linters:
+  enable:
+    - golint
```

Comment:
You should be able to just add staticcheck to this list instead of pulling in the staticcheck tool separately: https://github.com/golangci/golangci-lint#enabled-by-default-linters